kitchen.ec2.sh

```diff
@@ -63,7 +69,7 @@ then
   source "${THIS_DIR}/.kitchen.env.sh"
 fi
 
-if [ "$1" != "list" ]; then
+if [ "$1" == "create" ] || [ "$1" == "converge" ] || [ "$1" == "verify" ] || [ "$1" == "destroy" ] || [ "$1" == "test" ]; then
```

Comment:
nitpick: this does not work when a parameters are used as regex like`cr` instead of `create`

Reply:
Correct, thanks for spotting it.